APE10.rst

```diff
@@ -0,0 +1,178 @@
+Version      Release date        Notes
+==========   ==================  ==========================================
+v1.0 - LTS   February 19, 2015   Supported with bug fixes until June 2017
+v1.1         December 11, 2015
+v1.2         June 23, 2016
+v1.3         December 2016
+v2.0 - LTS   June 2017           Supported with bug fixes until end of 2019
+v3.0         December 2017       First release to support only Python 3+
+v3.1         June 2018
+v3.2         December 2018
+v3.3         June 2019
+v4.0 - LTS   December 2019       Supported with bug fixes for two years)
+==========   ==================  ==========================================
+
+Note that v1.0, v2.0, and v4.0 are marked as long-term support releases (LTS),
+which means that they are typically supported with bug fixes for at least two
+years. The above roadmap proposes that the last Python 2-compatible release will
+be v2.0, which we will support with bug fixes until the end of 2019. In the mean
+time, v3.0, to be released in December 2017, will be the first release that does
+not support Python 2.
+
+Note that v3.0 would not be an LTS release, since we would
+otherwise need to maintain two LTS releases in parallel. This is a one-off
+deviation from the version numbering policy outlined in `APE 2
+<https://github.com/astropy/astropy-APEs/blob/master/APE2.rst>`_.
+
+In order to decide which Python 3.x releases to support, we would follow the
+Python support calendar: the Python 3.x versions supported by an Astropy
+release would be the versions supported by the Python developers. Using the
+`current Python support calendar
+<https://docs.python.org/devguide/#status-of-python-branches>`_, this would
+mean that the following Astropy versions would support the following Python 3.x
+versions:
+
+==========    ===============================
+Version       Minimum required Python version
+==========    ===============================
+v2.0 - LTS    Python 2.7 or Python 3.3+
+v3.0          Python 3.4+
+v3.1          Python 3.4+
+v3.2          Python 3.4+
+v3.3          Python 3.5+
+v4.0 - LTS    Python 3.5+
+==========    ===============================
+
+Implementation
+--------------
+
+In practice, dropping Python 2.7 support will involve, amongst other things:
+
+* Removing the bundled `six <https://pythonhosted.org/six/>`_ package, and
+  updating all the code that used ``astropy.extern.six`` to use the Python 3
+  syntax directly.
+* Removing Python 2.7 from the configuration for the continuous integration
+  services (such as Travis and AppVeyor at the time of writing)
+* Making sure that the documentation on ReadTheDocs is built using Python 3.x
+* Removing any backports that may exist for Python 2.7 compatibility
+* Updating the documentation to remove any mention of Python 2.7, as well as
+  making sure that Python 3.x is not presented as a special case.
+* Removing Python 2 and 2.7 from the PyPI classifiers
```

Comment:
I'm not sure about this one: shouldn't it still be there because *past* versions will still support it?  Or are the classifiers distinct for each *version* as opposed to the whole package?